quick-start/src/main/java/com/marklogic/quickstart/service/DataHubService.java

```diff
@@ -186,7 +183,7 @@ private void installUserModules(HubConfig hubConfig, boolean forceLoad, DeployUs
         loadUserModulesCommand.setForceLoad(forceLoad);
         commands.add(loadUserModulesCommand);
 
-        SimpleAppDeployer deployer = new SimpleAppDeployer(hubConfig.getManageClient(), hubConfig.getAdminManager());
+        SimpleAppDeployer deployer = new SimpleAppDeployer(((HubConfigImpl)hubConfig).getManageClient(), ((HubConfigImpl)hubConfig).getAdminManager());
```

Comment:
I think we should be using HubConfig here, not HubConfigImpl.

Reply:
Exposes implementation of the manageclient & other mgmt library methods. Direction is to remove or encapsulate any dependencies that aren't directly supported.